pl_examples/intel_compressor/README.md

```diff
@@ -0,0 +1,82 @@
+# Step-by-Step
+
+This document describes the step-by-step instructions for reproducing PyTorch ResNet50/ResNet18/ResNet101 tuning results with [Intel® Neural Compressor](https://github.com/intel/neural-compressor).
+
+# Prerequisite
+
+### 1. Installation
+
+```shell
+pip install -r requirements.txt
```

Comment:
I would add and make intel neural compressor installation explicit here.

Reply:
Changed